### PR TITLE
UMI to AXI4-Lite converters and testbenches

### PR DESCRIPTION
**Pull request by azaidy**

### CODE REVIEW

File: umi/utils/testbench/tb_axilite2umi.v [hunk 1]
```diff
@@ -0,0 +1,363 @@
...
+    end
+
+    //always @(posedge clk) begin
+    //    if (axi_awvalid & axi_awready)
+    //        $display("Writing data to: 0x%h", axi_awaddr);
+    //    if (axi_wvalid & axi_wready)
+    //        $display("Data written: 0x%h, strobe: 0b%b", axi_wdata, axi_wstrb);
+    //    if (axi_arvalid & axi_arready)
+    //        $display("Reading data from: 0x%h", axi_araddr);
+    //end
```
Comment (gadfort) [inline umi/utils/testbench/tb_axilite2umi.v:288]: why is this here?

File: umi/utils/testbench/test_umi2axilite.py [hunk 2]
```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
```
Comment (gadfort) [inline umi/utils/testbench/test_umi2axilite.py:1]: fix testbench to match others

**Review by gadfort — CHANGES_REQUESTED** [review]

File: umi/utils/testbench/test_umi2axilite.py (hunk 2 above)
Comment (azaidy) [reply umi/utils/testbench/test_umi2axilite.py:1]: Resolved

File: umi/utils/testbench/tb_axilite2umi.v (hunk 1 above)
Comment (azaidy) [reply umi/utils/testbench/tb_axilite2umi.v]: Fixed!

File: umi/utils/rtl/axilite2umi.v [hunk 3]
```diff
@@ -0,0 +1,319 @@
...
+ *
+ ******************************************************************************/
+
+`default_nettype wire
+
+module axilite2umi #(
+    parameter CW    = 32,   // command width
+    parameter AW    = 64,   // address width
+    parameter DW    = 64,   // umi packet width
+    parameter IDW   = 16    // brick ID width
```
Comment (volkamir) [inline umi/utils/rtl/axilite2umi.v:29]: Better change this to 'UMI ID' or 'chip id' instead of 'brick ID' which is more specific.

**Review by volkamir — APPROVED** [review]

File: umi/utils/rtl/axilite2umi.v (hunk 3 above)
Comment (azaidy) [reply umi/utils/rtl/axilite2umi.v]: Resolved!

**Review by gadfort — APPROVED** [review]
